docs/components/props/Proptable.tsx

```diff
@@ -0,0 +1,136 @@
+import getConfig from "next/config";
+import { useState } from "react";
+import "./proptable.css";
+import Inline from "../code/Inline";
+import "nav-frontend-tabell-style";
+import { Select } from "nav-frontend-skjema";
+import Import from "../code/Import";
+
+const Proptable = ({
+  names,
+  defaultExport,
+  from,
+}: {
+  names: string[];
+  defaultExport?: string;
+  from: string;
+}) => {
+  const [tab, setTab] = useState(0);
+  const { publicRuntimeConfig } = getConfig();
+
+  const props = publicRuntimeConfig.props
+    .filter((comp) => [...names, defaultExport].includes(comp.displayName))
+    .map((comp) => {
+      if (comp.displayName === defaultExport) {
+        return { ...comp, defaultExport: true };
+      }
+      return { ...comp, defaultExport: false };
+    })
+    .sort((a, b) => a.defaultValue === b.defaultValue);
+
+  const headers = ["Prop", "Type", "Description"];
+
+  const ProptableHeader = (props) => <th key={props.index}>{props.val}</th>;
+
+  const parsePropValue = (val) => {
+    if (val && typeof val === "object") {
+      if (val.name === "enum") {
+        return val.value.map((x) => x.value).join(" | ");
+      }
+      val = val.name || val.value; // eslint-disable-line no-param-reassign
+    }
+    if (val === null || typeof val === "undefined" || val.length <= 0) {
+      return "-";
+    }
+    return val.toString();
+  };
+
+  const parseDescription = (desc) => {
+    const found = desc.match(/^@deprecated/);
+    if (found) {
+      return (
+        <span>
+          <strong>{found[0]}</strong>
+          {desc.substr(11, desc.length)}
+        </span>
+      );
+    }
+    return desc;
```

Comment:
Jeg er litt usikker på om dette funker, men kan du prøve noe sånt? (Parentesen i regex lager en "gruppe", og jeg lurer på om den gruppa blir lagt i `found`-arrayet)

```suggestion
    const found = desc.match(/^@deprecated(.*)/);
    if (found) {
      return (
        <span>
          <strong>{found[0]}</strong>
          {found[1]}
        </span>
      );
    }
    return desc;
```